app/views/users/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Name:</strong>
+  <%= @user.name %>
+</p>
+
+<p>
+  <strong>Email:</strong>
+  <%= @user.email %>
+</p>
+
+  <h2 class="table-head padding">My created events</h2>
+  <% @user.own_events.each do |e| %>
+    <p><b>Event:</b> <%= e.title %> <b>Date:</b> <%= e.date %></p>
+  <% end %>
+
+  <h2 class="table-head padding">My Past Events</h2>
+
+  <% past = user_clasify_events(@user.attendances, 'past') %>
```

Comment:
Not allowed in a views. I mean **affectations**
Take a look at this from `rails console`
```rb
2.7.0 :008 > u.events.upcoming_events
  Event Load (0.5ms)  SELECT "events".* FROM "events" INNER JOIN "attendances" ON "events"."id" = "attendances"."event_id" WHERE "attendances"."user_id" = ? AND (date >= '2020-09-13') LIMIT ?  [["user_id", 2], ["LIMIT", 11]]
 => #<ActiveRecord::AssociationRelation [#<Event id: 2, title: "Ores H Creator Vendri", body: "you ores pre abor creator", date: "2020-09-15", location: "Home", created_at: "2020-09-14 02:40:49", updated_at: "2020-09-14 02:40:49", creator_id: 2>]> 
2.7.0 :009 > 
2.7.0 :010 > 
2.7.0 :011 > User.all
  User Load (0.2ms)  SELECT "users".* FROM "users" LIMIT ?  [["LIMIT", 11]]
 => #<ActiveRecord::Relation [#<User id: 1, name: "pancho", email: "pancho@test.com", created_at: "2020-09-14 02:38:58", updated_at: "2020-09-14 02:38:59">, #<User id: 2, name: "Alice Bar", email: "alice@bar.com", created_at: "2020-09-14 02:39:58", updated_at: "2020-09-14 02:39:58">]> 
2.7.0 :012 > u = User.find(2)
  User Load (0.2ms)  SELECT "users".* FROM "users" WHERE "users"."id" = ? LIMIT ?  [["id", 2], ["LIMIT", 1]]
2.7.0 :013 > Attendance.all
  Attendance Load (0.3ms)  SELECT "attendances".* FROM "attendances" LIMIT ?  [["LIMIT", 11]]
 => #<ActiveRecord::Relation [#<Attendance id: 1, user_id: 1, event_id: 1, created_at: "2020-09-14 02:39:00", updated_at: "2020-09-14 02:39:00">, #<Attendance id: 2, user_id: 2, event_id: 2, created_at: "2020-09-14 02:41:03", updated_at: "2020-09-14 02:41:03">]> 
2.7.0 :014 > u.events.past_events
  Event Load (0.5ms)  SELECT "events".* FROM "events" INNER JOIN "attendances" ON "events"."id" = "attendances"."event_id" WHERE "attendances"."user_id" = ? AND (date < '2020-09-13') LIMIT ?  [["user_id", 2], ["LIMIT", 11]]
 => #<ActiveRecord::AssociationRelation []> 
2.7.0 :015 > u.events.upcoming_events
  Event Load (0.5ms)  SELECT "events".* FROM "events" INNER JOIN "attendances" ON "events"."id" = "attendances"."event_id" WHERE "attendances"."user_id" = ? AND (date >= '2020-09-13') LIMIT ?  [["user_id", 2], ["LIMIT", 11]]
 => #<ActiveRecord::AssociationRelation [#<Event id: 2, title: "Ores H Creator Vendri", body: "you ores pre abor creator", date: "2020-09-15", location: "Home", created_at: "2020-09-14 02:40:49", updated_at: "2020-09-14 02:40:49", creator_id: 2>]> 
2.7.0 :016 > 
```
That means that you can access to the upcoming attended events using
```rb
current_user.events.past_events
```